pennylane/templates/subroutines/multiplier.py

```diff
@@ -148,9 +147,9 @@ def __init__(
 
         self.hyperparameters["k"] = k
         self.hyperparameters["mod"] = mod
-        self.hyperparameters["work_wires"] = qml.wires.Wires(work_wires)
-        self.hyperparameters["x_wires"] = qml.wires.Wires(x_wires)
-        all_wires = qml.wires.Wires(x_wires) + qml.wires.Wires(work_wires)
+        self.hyperparameters["work_wires"] = work_wires
+        self.hyperparameters["x_wires"] = x_wires
+        all_wires = x_wires + work_wires
```

Comment:
the reason why I always translate to wires is to avoid problems like 1 + [2,3], in the case that the user passes an integer. Wouldn't this be a problem now?

Reply:
He's casting `work_wires` slightly earlier.